source/_integrations/proximity.markdown

```diff
@@ -20,95 +20,32 @@ Some examples of its use include:
 - Increase thermostat temperature as you near home
 - Decrease temperature the further away from home you travel
 
-The Proximity entity which is created has the following values:
-
-- `state`: Distance from the monitored zone (in `unit_of_measurement`)
-- `dir_of_travel`: Direction of the closest device or person to the monitored zone. Values are:
-  - `not set`
-  - `arrived`
-  - `towards`
-  - `away_from`
-  - `unknown`
-  - `stationary`
-- `unit_of_measurement`: Measurement of distance. Values are:
-  - `km`
-  - `m`
-  - `mi`
-  - `yd`
-  - `ft`
-- `nearest`: The device or person which is nearest to the zone
-
-To enable this integration in your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-proximity:
-  home: 
-    ignored_zones:
-      - work
-    devices:
-      - device_tracker.car1
-    tolerance: 50
-    unit_of_measurement: mi
-```
-
-### Video Tutorial
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The following sensor entities will be created
+
+### Distance
+
+For each tracked device or person, the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
+You can use the [Min/Max](/integrations/min_max) integration to determine the nearest and furthest distance.
+
+### Direction of travel
+
+For each tracked device or person, the direction of travel to or from the monitored zone. Values are:
+
+- `arrived`
+- `towards`
+- `away_from`
+- `stationary`
+
+### Nearest device
+
+The device or person which is nearest to the zone
+
+### Video tutorial
+
 This video tutorial explains how to set up geofencing in Home Assistant using the proximity integration.
 
 <lite-youtube videoid="pjAyRN5UiBg" videotitle="Geofencing in Home Assistant - Tutorial" posterquality="maxresdefault"></lite-youtube>
```

Comment:
Hi @smarthomejunkie
i saw you have created and added this tutorial (_#27427_) - first, many thanks for that, great video 👍 
Unfortunately this tutorial will not completely fit anymore after the implementation of the config flow (_[configuration via UI](https://github.com/home-assistant/core/pull/103894)_) and migration to common sensor entities (_https://github.com/home-assistant/core/pull/101497_). The way how to setup the integration as also the resulting entities and possibilities to automate based on these entities will change.
Therefore we need to consider to maybe remove this tutorial from the docs or at least add a warning that it is based on a deprecated version of this integration (_or even better and highly appreciated an adjusted tutorial will become available_ 😊 )
Nevertheless, many thanks for this great work 🙂

Reply:
Okay, time for a new video I guess ;)